src/analysis/clustering.jl

```diff
@@ -116,19 +118,18 @@ Hierarchically cluster \$S\$ scenarios with \$T\$ time steps each.
    https://doi.org/10.1007/s10618-013-0312-3
 """
 function cluster_series(
-    data::AbstractMatrix{T},
-    n_clusters::Int64
-)::Vector where {T<:Real}
+    data::AbstractMatrix{T}, n_clusters::Int64
+)::Vector{Int64} where {T<:Real}

```

Comment:
```suggestion
    data::AbstractMatrix{<:Real}, n_clusters::Int64
)::Vector{Int64}
```

Remove `T` for conciseness as it is only used once